set-java-home.bash

```diff
@@ -8,6 +8,10 @@ asdf_update_java_home() {
 }
 
 prompt_command() {
+  if [ "$PWD" == "$LAST_PWD" ]; then
```

Comment:
Please use `[[` and `]]`, also please surround expanded variables with `{}` - e.g. "${PWD}"